package.json

```diff
@@ -5,6 +5,13 @@
   "main": "./dist/cjs/i18next.js",
   "module": "./dist/esm/i18next.js",
   "types": "./index.d.ts",
+  "exports": {
```

Comment:
is this whole section still required?

Reply:
I think so... I guess I could remove the `"."` section, and the `types` are for future-proof `typescript` which will support the types key in there: https://www.typescriptlang.org/docs/handbook/esm-node.html